data_linked_in_ssr_manual/loudspeakers_1-2.asd

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<asdf>
+  <header>
+    <name>.</name>
```

Comment:
Does this mean anything?
Doesn't it work without a `<name>`?

Reply:
Removed